src/App.jsx

```diff
@@ -1,13 +1,84 @@
+import {useState} from "react";
+import "./App.css";
 
-import './App.css'
+import FruitArticle from "./components/fruitAtricle/FruitArticle.jsx";
+import InputField from "./components/inputField/InputField.jsx";
+import SelectField from "./components/selectField/SelectField.jsx";
+import RadioField from "./components/radioField/RadioField.jsx";
+
+import strawberryIcon from './assets/strawberry.png';
+import bananaIcon from './assets/banana.png';
+import appleIcon from './assets/apple.png';
+import kiwiIcon from './assets/kiwi.png';
+import CheckField from "./components/checkField/CheckField.jsx";
 
 function App() {
 
-  return (
-    <>
-        <h1>Fruitmand bezorgservice</h1>
-    </>
-  )
+    const [strawberryCount, setStrawberryCount] = useState(0);
+    const [bananaCount, setBananaCount] = useState(0);
+    const [appleCount, setAppleCount] = useState(0);
+    const [kiwiCount, setKiwiCount] = useState(0);
```

Comment:
je had hier     const [fruitCount, setFruitCount] = useState({
        strawberry: 0,
        banana: 0,
        apple: 0,
        kiwi: 0
    }); 
    
    Kunnen gebruiken, echter moet er dan nog wel wat veranderen in de rest van de code :)